src/styles/blocks/card.scss

```diff
@@ -0,0 +1,122 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  padding: 32px 16px 16px;
+  width: $card-width;
+  border-radius: 5px;
+  border: 1px solid $border-card-color;
+
+  background-color: $background-card-color;
+
+  &:hover {
+    transform: scale(1.2);
+    transition: transform $hover-transition-duration;
```

Comment:
don't add `transition` property on hover...cause it doesn't work correct

you can check this...mouse over the card and then take it away, and you will see that the card returns to its starting position very quickly and sharply

it shouldn't be like that

Reply:
Ok, but in condition we have "Make all the changes smooth on hover (during 300ms):", not both ways - only to one.